ci/wait.yaml

```diff
@@ -0,0 +1,15 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: alpine
+inputs:
+  - name: eq-questionnaire-runner
+params:
+  RUNNER_FQDN:
+run:
+  path: sh
+  args:
+    - -exc
+    - |
+      until wget -qO- https://${RUNNER_FQDN}/status 2>&1 | grep "\"status\": \"OK\""; do sleep 5; done
```

Comment:
Previously this checked the version on the status page so that it waited for the requested deployed version.